draft-ietf-tls-tls13.md

```diff
@@ -5111,6 +5100,19 @@ also provide collision resistance.
 Note: The binder does not cover the binder values from other
 PSKs, though they are included in the Finished MAC.
 
+Note: TLS does not currently permit the server to send a certificate_request
+message with non-certificate-based handshakes (e.g., PSK).
+If this restriction were to be relaxed in future, the
+client's signature would not cover the server's certificate directly.
+However, if the PSK was established through a NewSessionTicket, the client's
+signature would transitively cover the server's certificate through
+the PSK binder. {{PSK-FINISHED}}
+describes a concrete attack on constructions that do not bind to
+the server's certificate. It is unsafe to use certificate-based client
+authentication when the client might potentially share the same
+PSK/key-id pair with two different endpoints and implementations
+MUST NOT combine external PSKs with certificate-based authentication.
```

Comment:
I guess I've forgotten whether Russ just wanted to be able to do cert auth with NST-provisioned PSKs or with external PSKs as well.  If the latter, to double-check, we're happy with some future document overriding this MUST NOT?